src/map/Pins.js

```diff
@@ -74,17 +75,22 @@
 
     Pins.prototype.layerEnter = function (base, svgElement, domElement) {
         Layer.prototype.layerEnter.apply(this, arguments);
-
+        
         this._pinsTransform = svgElement;
         this._selection.widgetElement(this._pinsTransform);
         this.pinsPaths = d3.select(null);
+        this
+            .tooltipHTML(function (d) {
+                return d.ext.tooltip === undefined ? "" : d.ext.tooltip;
+            })
+        ;
     };
 
     Pins.prototype.layerUpdate = function (base) {
         Layer.prototype.layerUpdate.apply(this, arguments);
 
         this._pinsTransform
-            .style("opacity", this.opacity())
+        .style("opacity", this.opacity())
```

Comment:
@GordonSmith These indentation changes are somehow happening via VSCode. Have you experienced this issue before?

Reply:
FWIW I turn off "format on save" for non VS Code projects (File/settings -> workspace).  Once you get to 2.0, they have all been reformatted for vscode.

Also you can add the ?w=1 option to the address bar to ignore the whitespace changes.

Reply:
Which address bar? The git repo I'm pointing to while pushing?

Reply:
The one pointing to this page ^^^^